ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplDumpTask.java

```diff
@@ -196,14 +196,14 @@ private Long bootStrapDump(Path dumpRoot, DumpMetaData dmd, Path cmRoot) throws
       Path dbRoot = dumpDbMetadata(dbName, dumpRoot);
       dumpFunctionMetadata(dbName, dumpRoot);
 
-      String uniqueKey = Utils.setDbBootstrapDumpState(hiveDb, dbName);
+      String uniqueKey = Utils.setDbBootstrapDumpState(hiveDb, dbName, LOG);
```

Comment:
dont need to pass the LOG in create a new instance in the Utils class.

Reply:
Done